apps/ern-skin/src/components/Publications.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <MessageBox type="error" v-if="error" class="publication-list-error">
+    <p><strong>Unable to retrieve publications</strong></p>
+    <p>{{ error }}</p>
+  </MessageBox>
+  <MessageBox
+    type="error"
+    v-else-if="!data.length && !error"
+    class="publication-list-error"
+  >
```

Comment:
The error message is shown very briefly when the publications element is expanded. I think we can combine these two error messages.

Reply:
Ah, there's an extra exclamation point in error-

```diff
- v-else-if="!data.length && !error"
+ v-else-if="!data.length && error"
```